cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
That being said, we might have explicitly enabled a warning that requires every field to be explicitly initialized. However, since newer versions of gcc don't produce this warning, the gcc developers seem to have decided that this was indeed a mistake.